hedera-bls-api/src/main/java/com/hedera/platform/bls/api/Field.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.platform.bls.api;
+
+/**
+ * Interface representing a generic field
+ *
+ * <p>This is a factory interface, responsible for creating {@link FieldElement field elements}
+ */
+public interface Field {
+    /**
+     * Creates a new field element from a long
+     *
+     * @param inputLong the long to use to create the field element
+     * @return the new field element
+     */
+    FieldElement elementFromLong(final long inputLong);
```

Comment:
nit, `final` is not needed in interfaces for methods that don't have implementations. (Still use them for default methods though.)
```suggestion
    FieldElement elementFromLong(long inputLong);
```

Reply:
fixed throughout